### DISCUSSION TITLE
throat.inscribed_diameter is larger than the pore.inscribed_diameter

### DISCUSSION THREAD

**Discussion by Devin-one** [PMEAL/porespy#880] [Q&A]
Hello professor. I extracted a three-dimensional pore network using ImageJ and SNOW algorithms. But I found that the `throat.inscribed_diameter ` is larger than the `pore.inscribed_diameter` . What are the reasons for this?

**Comment by jgostick** [comment]
This is due to pores which are 'flat'. This causes the inscribed sphere to be limited by the length of the flat dimension.  The spheres drawn in throats on the other hand can extend into both the neighboring pores so have a lot more room.  The solution to this problem is to use 'cones_and_cylinders' geometry in OpenPNM, since this can handle larger throats.   I think this question is common enough that I should write an example about it, and add it to the list of FAQs.
